thyrel-web/src/components/room/book/BookSentenceElement.tsx

```diff
@@ -34,12 +39,9 @@ export default function BookSentenceElement({
       <Box
         display="flex"
         flexDirection="row"
-        alignItems="center"
-        p={1}
-        border={1}
-        borderColor="primary.main"
-        borderRadius={4}
-        maxWidth={336}>
+        p={0.5}
+        width={320}
+        className={classes.box}>
         <Avatar src={avatarUrl} className={classes.avatar} />
         <Box ml={1} mr={1}>
```

Comment:
pour fixé le bug que j'ai screen il faut utiliser `align-self` je pense
```suggestion
        <Box ml={1} mr={1} alignSelf="center">
```

Reply:
Donc ça serait ça pour faire aller à la ligne le texte si trop long?

Reply:
non c'est pour aligné le text horizontalement par rapport à ce message https://github.com/GinierClasses/holydraw/pull/310#pullrequestreview-648034552